pkg/pillar/diskmetrics/usage.go

```diff
@@ -201,3 +203,48 @@ func PersistUsageStat(log *base.LogObject) (*types.UsageStat, error) {
 	}
 	return usageStat, nil
 }
+
+// FoundLargeFiles is returned by FindLargeFiles
+type FoundLargeFiles struct {
+	Path string
+	Size int64
+}
+
+// FindLargeFiles walks a directory and reportes all large files unless
```

Comment:
Just curious, "reportes" is a valid form or typo? (or more like a French? :)

Reply:
typo. fixed